persistence/javadsl/src/main/scala/com/lightbend/lagom/javadsl/persistence/PersistenceModule.scala

```diff
@@ -3,64 +3,18 @@
  */
 package com.lightbend.lagom.javadsl.persistence
 
-import com.google.inject.{ AbstractModule, Inject, TypeLiteral }
-import com.google.inject.matcher.AbstractMatcher
-import com.google.inject.spi.InjectionListener
-import com.google.inject.spi.TypeEncounter
-import com.google.inject.spi.TypeListener
-import com.lightbend.lagom.javadsl.api.ServiceLocator
-import com.lightbend.lagom.javadsl.persistence.PersistenceModule.InitServiceLocatorHolder
-import akka.actor.ActorSystem
+import com.google.inject.AbstractModule
 import com.lightbend.lagom.internal.javadsl.persistence.{ ReadSideConfigProvider, ReadSideImpl }
-import com.lightbend.lagom.internal.persistence.{ ReadSideConfig, ServiceLocatorHolder }
+import com.lightbend.lagom.internal.persistence.ReadSideConfig
 
 /**
  * Guice module for the Persistence API.
  */
 class PersistenceModule extends AbstractModule {
   override def configure(): Unit = {
-    binder.bind(classOf[PersistenceModule.InitServiceLocatorHolder]).asEagerSingleton()
```

Comment:
these things were moved to `CassandraPersistenceModule`, since we only use `ServiceLocator` for Cassandra